src/program/lwaftr/tests/data/icmp_endaddr.conf

```diff
@@ -42,27 +42,37 @@ softwire-config {
     error-rate-limiting {
       packets 600000;
     }
-    ip 10.10.10.10;
-    mac 12:12:12:12:12:12;
     mtu 2000;
-    next-hop {
-      mac 68:68:68:68:68:68;
-    }
     reassembly {
       max-fragments-per-packet 40;
     }
   }
+  instance {
+    device test;
```

Comment:
We actually discussed using `name` as the key a while ago and decided against it. I think if I remember correctly we didn't want to do that because the name is not a per instance name but rather per collection of instances. For example you'd want to be able to do:

```
$ snabb config get mylwaftr /
```

This should find the collection of instances with the name `mylwaftr` and displaying all of them, if that was the key in an instance surely it'd only display that single instance. I also don't know if we're wanting specific `internal-interface` and `external-interface` settings per queue (VLAN tag or whatever) but with the schema above that wouldn't be possible. What would be the use of the `queue` list in the above schema?